### DISCUSSION TITLE
Experimental Autopilot tempremental

### DISCUSSION THREAD

**Discussion by scotgr** [flybywiresim/aircraft#3354] [Help]
Sorry if this has been covered or if no support is available.  With the experimental version I find that the vertical navigation doesnt always work.  When I take off engage autopilot it wont follow the flight directors but if I quit the flight and start again 1 or 2 times it works.  I wondered if there is a knack to getting it to work first time or something I am doing wrong?

Thanks :)

**Comment by robinottawa** [comment]
Tonight I could not get it to fly on AP direct to a waypoint on the flight plan. Didn't notice if vertical climb was also affected. Latest update.

**Reply by scotgr** [reply]
I’ve not had a problem with that but you would notice the vnav right away. It won’t hold an altitude, climb or descend

**Reply by robinottawa** [reply]
Same flight today with no update, and vertical is fine. And AP seems fine too. So, good luck.

**Reply by scotgr** [reply]
I updated last night after my flight and it worked first time today. I am going to try again later 😊

**Reply by omarsmak** [reply]
Yeah sometimes you will need to load the plane few times until it works

**Comment by riveroalbert** [comment]
I'm pretty sure direct waypoint routing is still not 100% implemented, same as holds. I've also experienced strange behaviour but be patient, the developers are aware and working on it!
